src/Swarm/TUI/View.hs

```diff
@@ -217,17 +217,24 @@ drawNewGameMenuUI (l :| ls) =
     let bestRealTime = if oneBest then "best:" else "best real time:"
     let noSame = if oneBest then const Nothing else Just
     let lastText = let la = "last:" in padRight (Pad $ T.length bestRealTime - T.length la) (txt la)
-    vBox . catMaybes $
-      [ Just $ txtWrap (nonBlank (s ^. scenarioDescription))
-      , Just $
-          padTop (Pad 3) $
-            padRight (Pad 1) (txt bestRealTime) <+> describeStatus (si ^. scenarioBestTime)
-      , noSame $ -- hide best game time if it is same as best real time
-          padTop (Pad 1) $
-            padRight (Pad 1) (txt "best game time:") <+> describeStatus (si ^. scenarioBestTicks)
-      , Just $
-          padTop (Pad 1) $
-            padRight (Pad 1) lastText <+> describeStatus (si ^. scenarioStatus)
+    vBox
+      [ txtWrap (nonBlank (s ^. scenarioDescription))
+      , vBox . catMaybes $
+          [ padTop (Pad 1)
+              . padRight (Pad 1)
+              . (txt "Author: " <+>)
+              . txt
+              <$> (s ^. scenarioAuthor)
```

Comment:
The author field could be coloured differently, as it's only one line from the description and kind of melds with it.

I would suggest the [`dim`](https://hackage.haskell.org/package/vty-5.37/docs/Graphics-Vty-Attributes.html#v:dim) attribute. :slightly_smiling_face: